plugins/module_utils/specs.py

```diff
@@ -68,8 +68,15 @@ def op_item_info():
             type="str",
             required=True
         ),
+        flatten_fields_by_label=dict(
+            type="bool",
+            default=True
```

Comment:
even if the default behaviour will be returning the fields as a list starting from 3.0.0, we will still be allowing users to set `flatten_fields_by_label` to `true`, in order to get a dictionary if this is the desired behaviour, right?

Reply:
Good point! I agree we should keep the option around in v3.0.0 and instead switch the default to `default=False`